projects/freetype2/Dockerfile

```diff
@@ -20,6 +20,7 @@ RUN apt-get install -y make autoconf libtool libarchive-dev
 
 # Get some files for the seed corpus
 ADD https://github.com/adobe-fonts/adobe-variable-font-prototype/releases/download/1.001/AdobeVFPrototype.otf $SRC/font-corpus/
+RUN git clone https://github.com/unicode-org/text-rendering-tests.git && cp text-rendering-tests/fonts/* $SRC/font-corpus
```

Comment:
You can do ADD. See above. Supposedly it could check for updates during rebuild.